CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Piral Changelog
 
-## 0.12.2 (tbd)
+## 1.0.0 (tbd)
```

Comment:
This is unfortunate, because it would show `1.0.0` in the documentation. Can you get the `master` branch (instead of `develop`) combined with your changes into `documentation`?

Reply:
adapted